pkg/apiclient/apiclient.go

```diff
@@ -32,7 +32,7 @@ type Opts struct {
 	// DEPRECATED: use `ClientConfigSupplier`
 	ClientConfig         clientcmd.ClientConfig
 	ClientConfigSupplier func() clientcmd.ClientConfig
-	Offline              bool
```

Comment:
this is a breaking change, can we keep the old variable please

Reply:
I put it back. It should still work as it did before for workflows. But now, having an empty `OfflineFiles` will fail when trying to lint a workflowtemplate that has a ref